instagram/src/components/Login/Login.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./login.css";
+
```

Comment:
For the login page try using using state in a class component. Then you can have a handler for change in input and a handler for submitting the login.